installation/install.sh

```
@@ -41,6 +41,7 @@ KYLIN_V10_OS="ky10sp1 ky10sp2 ky10sp3 ky10sp3.2403"
 XINCHUANG_OS="$KYLIN10_OS uos20"
 SUPPORTED_OS="$REDHAT_OS $DEBIAN_OS"
 REDHAT_WITHOUT_CENTOS6=`echo $REDHAT_OS |sed s/CENTOS6//`
+SUPPORTED_LOCALE=("en_US" "zh_CN")
```

Shellcheck (0.10.0)

[warning] SUPPORTED_LOCALE appears unused. Verify use (or export if used externally).

(SC2034)